creator/nx_creator_xpcs.py

```diff
@@ -78,6 +78,47 @@ def _create_dataset(self, group, name, value, **kwargs):
         ds.attrs["target"] = ds.name
         return ds
 
+    def _check_unit(self, name, expected, supplied):
+        """
+            Our test for units should check if the supplied
+            units string can be mapped into the expected units for that field.
+            If arbitrary units are supplied in form of 'au', 'a.u.' or 'a.u' no conversion is applied
+            and pint if not used for the unit check.
+
+        :param : name of field
+        :param : expected units
+        :param : units string that was given
+        :return *bool*: `True` if units conversion is possible:
+        """
+
+        # catch arbitrary unit separately from pint --> point that out in documentation
+        if supplied == 'au' or 'a.u.' or 'a.u':
```

Comment:
I think this syntax should be:
```py
if supplied in ["au", "a.u.", "a.u"]:
```
The current implementation will default to True even if supplied doesn't match any of the options.